docgen/layouts/common/meta.pug

```diff
@@ -24,6 +24,6 @@ html
     meta(content='Boost search on your React apps with Algolia', property='og:description')
     meta(content='React InstantSearch', property='og:site_name')
 
-    title React InstantSearch | #{title}
+    title React InstantSearch | Lightning Fast React and React Native Search
```

Comment:
So we're clear here, this means that all the pages will have the same title. Is that what you want @neilrichler